app.js

```diff
@@ -23,6 +23,15 @@ var pool = mysql.createPool({
     port            : process.env.RDS_PORT
 });
 
+// var pool = mysql.createPool({
```

Comment:
Don't just comment this out.  Delete this.  The password has been committed to a public source code repository and is publicly discover-able.

We also need to change the password.

Reply:
Fixed this